packages/api/pw-test.config.cjs

```diff
@@ -11,54 +11,34 @@ dotenv.config({
 })
 
 const cli = path.join(__dirname, 'scripts/cli.js')
-
 /** @type {import('esbuild').Plugin} */
 const nodeBuiltinsPlugin = {
   name: 'node builtins',
   setup(build) {
     build.onResolve({ filter: /^stream$/ }, () => {
       return { path: require.resolve('readable-stream') }
     })
-
-    build.onResolve({ filter: /^cross-fetch$/ }, () => {
-      return { path: path.resolve(__dirname, 'scripts/fetch.js') }
-    })
   },
 }
 
 /** @type {import('playwright-test').RunnerOptions} */
 module.exports = {
   buildConfig: {
-    inject: [path.join(__dirname, './scripts/node-globals.js')],
     plugins: [nodeBuiltinsPlugin],
     define: {
       DATABASE_URL: JSON.stringify(process.env.DATABASE_URL),
       DATABASE_TOKEN: JSON.stringify(process.env.DATABASE_TOKEN),
     },
   },
   buildSWConfig: {
-    inject: [
-      path.join(__dirname, './scripts/node-globals.js'),
-      path.join(__dirname, './test/scripts/worker-globals.js'),
-    ],
     plugins: [nodeBuiltinsPlugin],
     define: {
-      DATABASE_URL: JSON.stringify(process.env.DATABASE_URL),
+      DATABASE_URL: JSON.stringify(process.env.PW_DATABASE_URL),
```

Comment:
@redaphid I don't see `PW_DATABASE_URL` defined anywhere. maybe this is left over from local debugging?

Reply:
ah. Yeah we can remove all these defines now. Good catch!